portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,5 +31,13 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
```

Comment:
you don't need to set the content type to HTML if you are planning to return json. there is no way to set the content type twice. My recommendation is to just return all the data as json and then perform an html formatting/rendering you need post-fetch.